README.md

```diff
@@ -76,6 +76,7 @@ Finally, we also loaded the pre-specified hostmap configuration from file using.
 # load host map configuration from file
 config = tfe.config.load('config.json')
 ```
+Take a look at [`/tools/gcp/link`](https://github.com/mortendahl/tf-encrypted/blob/master/tools/gcp/link) as an example to generate the config file for gcp. If you run it locally, you can use simply `tfe.LocalConfig`. You can find an example [here](https://github.com/mortendahl/tf-encrypted/blob/doc/examples/federated-average/run.py#L20).
```

Comment:
fyi urls can be relative, ie `./tools/gcp/link` instead of `https://github.com/mortendahl/tf-encrypted/blob/master/tools/gcp/link` -- this would make the link work on non-master branches as well